django/__init__.py

```diff
@@ -1,6 +1,6 @@
 from django.utils.version import get_version
 
-VERSION = (5, 2, 0, "alpha", 0)
```

Comment:
The commit message here can say `main` (if not already on your radar).

Reply:
Thank you :+1: